index.html

```diff
@@ -1651,7 +1651,14 @@ <h1>Friends and Partners</h1>
  								<li><a target="_self" href="https://gitlab.com/fossasia"><i class="fab fa-gitlab"></i></a></li>
 								<li><a target="_self" href="https://gitter.im/fossasia"><i class="fab fa-gitter"></i></a></li>
 								<li><a target="_self" href="https://de.wikipedia.org/wiki/FOSSASIA"><i class="fab fa-wikipedia-w"></i></a></li>
+           <li>
 
+<a target="_self" href="https://www.crunchbase.com/organization/fossasia">
+   <img style="width:20px;height:20px" src="http://gecad.com/wp-content/themes/gecad/assets/images/icon-crunchbase-black.svg"/>     
```

Comment:
Instead of using an svg, please use an icon provided by an imported icon pack (font awesome?)